components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -144,9 +145,10 @@
   // #endregion mount and prop initialization
 
   function saveProperties() {
-    console.log("Saving the following properties:");
-    Object.entries(_this).forEach(([k, v]) => {
-      console.log(k, v);
+    saveManifest({
+      id,
+      access_token,
+      manifest: { settings: JSON.parse(_this) },
```

Comment:
JSON.parsing `_this` here and would like to not have to do that in the future -- but if I don't, it tries to save the proxy, which is stringified upon XHR.

Reply:
Wasn't this Oliver's suggested way of doing this? 🤔

```suggestion
      manifest: { settings: { ..._this } },
```